### PR TITLE
Fix/extensions components

### PR DESCRIPTION
**Pull request by NikolaL87**

### CODE REVIEW

File: lib/Title/custom-title.scss [hunk 1]
```diff
@@ -2,44 +2,51 @@
...
+		left: 0;
+		top: 0;
+		&-puzzle {
+			background: url('../../img/icons/icon-title-puzzle.png') no-repeat 50%;
+		}
+		&-object {
+			background: url('../../img/icons/icon-title-object.png') no-repeat 50%;
+		}
+		& + .custom-title-label {
       padding-left: 56px;
```
Comment (EdytaSki) [inline lib/Title/custom-title.scss]: keep a good indentation please

File: src/FileUpload/index.js [hunk 2]
```diff
@@ -56,7 +57,7 @@ class FileUpload extends Component {
               <div className="container__col-xs-6 center-vertical">
```
Comment (EdytaSki) [inline src/FileUpload/index.js]: Pay attention to alignment styles

File: src/Title/custom-title.scss [hunk 3]
```diff
@@ -2,44 +2,51 @@
...
+		left: 0;
+		top: 0;
+		&-puzzle {
+			background: url('../../img/icons/icon-title-puzzle.png') no-repeat 50%;
+		}
+		&-object {
+			background: url('../../img/icons/icon-title-object.png') no-repeat 50%;
+		}
+		& + .custom-title-label {
       padding-left: 56px;
```
Comment (EdytaSki) [inline src/Title/custom-title.scss]: please make corrections on alignment styles

File: src/Title/index.js [hunk 4]
```diff
@@ -1,12 +1,15 @@
 import React from "react";
 import "./custom-title.scss";
 
-const Title = ({ icon, label }) => (
-  <h2 className="custom-title">
+const Title = ({ icon, label, titleColor , onClick}) => (
```
Comment (EdytaSki) [inline src/Title/index.js]: Please explain in few words what this part of code should do?

File: stories/Title.js [hunk 5]
```diff
@@ -10,3 +10,9 @@ storiesOf("Title", module).add(
   () => <Title label="Test" icon="object" />,
   { notes: "A very simple component" }
 );
+storiesOf("Title", module).add(
+  "Title - custom blue title with icon",
+  () => <Title label="Test" icon="puzzle" titleColor="blue" />,
```
Comment (EdytaSki) [inline stories/Title.js]: It's acceptable - we will define these rules later in details.

File: lib/Title/custom-title.scss (hunk 1 above)
Comment (NikolaL87) [reply lib/Title/custom-title.scss]: Code inside of the lib is formatted automatically.

File: src/Title/index.js (hunk 4 above)
Comment (NikolaL87) [reply src/Title/index.js]: You have a props that you pass to the Title component.

File: stories/Title.js (hunk 5 above)
Comment (NikolaL87) [reply stories/Title.js]: Ok.

File: src/Title/custom-title.scss (hunk 3 above)
Comment (NikolaL87) [reply src/Title/custom-title.scss]: On my code editor, everything is aligned properly.

File: src/FileUpload/index.js (hunk 2 above)
Comment (NikolaL87) [reply src/FileUpload/index.js]: Fixed.

File: lib/Title/custom-title.scss (hunk 1 above)
Comment (EdytaSki) [reply lib/Title/custom-title.scss]: Please make corrections on lines 36, 36 the code is not aligned properly.

File: src/Title/custom-title.scss (hunk 3 above)
Comment (EdytaSki) [reply src/Title/custom-title.scss]: Line 36 - the commited code should be aligned please

File: src/Title/index.js (hunk 4 above)
Comment (EdytaSki) [reply src/Title/index.js]: Please add the explicit comment into the code : 
example ;
//You have a props that you pass to the Title component.
const Title = ({ icon, label, titleColor , onClick}) => (

File: lib/Title/custom-title.scss (hunk 1 above)
Comment (NikolaL87) [reply lib/Title/custom-title.scss]: indentation
Which correction? The code is formatted well.

File: src/Title/index.js (hunk 4 above)
Comment (NikolaL87) [reply src/Title/index.js]: That is basic stuff in React. If we start to put comments on these things we will have a lot of unnecessary comments about those obvious things. That is my personal opinion.

**Pull request merged**